src/views/Home.jsx

```diff
@@ -1,8 +1,38 @@
 import './Home.css';
 import { SingleList } from '../components';
+import { useState } from 'react';
+import { createList, useAuth } from '../api';
+import { useNavigate } from 'react-router-dom';
 
 export function Home({ data, setListPath }) {
 	const hasList = data.length !== 0;
+
+	const { user } = useAuth();
+	const userId = user?.uid;
+	const userEmail = user?.email;
+
+	const [inputValue, setInputValue] = useState('');
+	const navigate = useNavigate();
```

Comment:
praise: I really like the use of `useNavigate`! I've never actually used it in a project and while I understand it theoretically when reading about it. It's awesome to see it in use here!